examples/01_introduction/01_Hello world.md

```diff
@@ -10,7 +10,9 @@ fun main() {                        // 2
 
 1. Kotlin code is usually defined in packages. Package specification is optional: If you don't specify a package in a source file, its content goes to the default package.
 2. An entry point to a Kotlin application is the `main` function. In Kotlin 1.3, you can declare `main` without any parameters. The return type is not specified, which means that the function returns nothing. 
-3. `println` writes a line to the standard output. It is imported implicitly. Also note that semicolons are optional.
+3. `println` and `print` writes a line to the standard output with cursor ending no new line and in same line respectively. It is imported implicitly.
```

Comment:
In all the "Kotlin by Example" lessons we try to keep explanations as short as possible and explain only what happens in the code. For more detailed info, we offer the documentation.
This example shows how easy it is to write "Hello world" in Kotlin. I wouldn't overload it with details irrelevant for this exact line of code (no `print` function in it).